.isort.cfg

```diff
@@ -2,7 +2,6 @@
 line_length=120
 multi_line_output=0
 lines_after_imports=2
-add_imports=from __future__ import unicode_literals
```

Comment:
there are still quite a few older files that don't use unicode literals and getting warnings (or now: travis failures) there is pretty annoying...